tests/models/test_submissions.py

```diff
@@ -25,7 +21,7 @@ def test_submission_model_str(self):
         submission = mommy.make(
             'tasking.Submission',
             task=cattle,
-            _fill_optional=['user', 'comment', 'submission_time'])
```

Comment:
What are the effects of changing from `comment` to `comments`?

Reply:
The `comment` field doesn't exist for the Submission model.